### ISSUE TITLE
call OHLC data for only the past hour of a pair

### ISSUE THREAD

**Issue by Jens1989** [ccxt/ccxt#5143]
- OS: windows 10
- Programming Language version: python 3.7.3
- CCXT version: 1.18.504
- method: fetch_ticker

I am trying to execute a strategy which would take trades on an hourly basis. I'm first calling the historical data for a pair and then when the new hour starts I Just want to have the latest OHLC data for the past hour to speed up computational time. 

Looking in the manual I found the fetch_ticker method with option params. I haven't been able to figure out what / how to use the params. when I run the following (where start and end would be the unix time for the past hour, although that's irrelevant as I'm getting an unrelated error) I'm getting an error

`exchange.fetch_ticker('XRP/BTC', params ={'openTime':start,'closeTime':end})`

`ExchangeError: exchange{"code":-1101,"msg":"Too many parameters; expected '1' and received '3'."}`

**Comment by kroitor** [comment]
When submitting an issue it is highly recommended to follow these guides step-by-step: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue precisely ) You forgot to mention which exchange it is... However, from the error message it looks like it's Binance.

> I haven't been able to figure out what / how to use the params.

Using the overrides with extra parameters is described in the Manual here:

- https://github.com/ccxt/ccxt/wiki/Manual#overriding-unified-api-params

Those custom params are exchange-specific. To use those params, you should:

1. Open the docs for the exchange method in question: https://github.com/binance-exchange/binance-official-api-docs/blob/master/rest-api.md#24hr-ticker-price-change-statistics (to see which exchange endpoint it is use the `verbose` mode as shown in the [examples](https://github.com/ccxt/ccxt/tree/master/examples))

2. Note the set of parameters that are accepted: there are no `openTime` or `closeTime` parameters there ) You should only use those if you really know what you're doing.

The `fetchTicker` call in general is used to retrieve 24-hour stats, so you can't specify the timerange for it. In order to get OHLCV data, you should use a different method, that is `fetchOHLCV`, also described in the Manual here: https://github.com/ccxt/ccxt/wiki/Manual#ohlcv-candlestick-charts, and [examples](https://github.com/ccxt/ccxt/tree/master/examples) as well.

In order to specify the timerange for `fetchOHLCV` using the `params` overrides is not necessary either – better use the unified pagination as described in the Manual here: https://github.com/ccxt/ccxt/wiki/Manual#pagination

Basically, I'd recommend to read the entire Manual from top to bottom at least once in order to understand how to use this library efficiently. That will help save you a ton of time, trust me )

Hope that answers the question, let us know if not.
